be/src/tools/meta_tool.cpp

```diff
@@ -103,6 +106,10 @@ DEFINE_string(tablet_file, "", "file to save a set of tablets");
 DEFINE_string(file, "", "segment file path");
 DEFINE_int32(column_index, -1, "column index");
 DEFINE_int32(key_column_count, 0, "key column count");
+DEFINE_int64(expired_sec, 86400, "expired seconds");
+DEFINE_string(conf_file, "", "conf file path");
+DEFINE_string(audit_file, "", "audit file path");
+DEFINE_bool(do_delete, false, "do delete files");
```

Comment:
rename to dry_run? which might be easier to understand.

Reply:
`do_delete` do not mean dry run, the tool only delete files when `do_delete` is true, otherwise just output a list of files that may need to be deleted